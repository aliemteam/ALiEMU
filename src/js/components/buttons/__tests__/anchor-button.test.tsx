import React from 'react';
import renderer from 'react-test-renderer';

import AnchorButton from '../anchor-button';

it('should render correctly', () => {
    const tree = renderer.create(
        <>
            <AnchorButton>Hello World</AnchorButton>
        </>,
    );
    expect(tree).toMatchSnapshot();
});
