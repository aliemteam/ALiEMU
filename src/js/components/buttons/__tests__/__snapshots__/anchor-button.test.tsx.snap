// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<a
  className="btn"
  role="button"
  tabIndex={0}
>
  Hello World
</a>
`;
